compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeSceneMediator.uikit.kt

```diff
@@ -118,12 +124,51 @@ internal class ComposeSceneMediator(
         )
     }
 
+    private val semanticsOwnerListener: PlatformContext.SemanticsOwnerListener by lazy {
+        object : PlatformContext.SemanticsOwnerListener {
+            var current: Pair<SemanticsOwner, AccessibilityMediator>? = null
+
+            override fun onSemanticsOwnerAppended(semanticsOwner: SemanticsOwner) {
+                if (current == null) {
+                    current = semanticsOwner to AccessibilityMediator(viewController.view, semanticsOwner, coroutineContext)
+                } else {
+                    // Multiple SemanticsOwner`s per ComposeSceneMediator is a legacy behavior and will not be supported
+                    // TODO: refactor PlatformContext when legacy behavior is removed altogether
```

Comment:
This behavior won't be removed from `PlatformContext`, it will be "legacy" at some only for some platforms such as iOS.

So if you have some points to refactor `PlatformContext` let's discuss it now

Reply:
That means that implementation details of other platforms leaked into the common interface.

Reply:
multi owner scene is not other platform details, it's one of the common implementations. Yes, I think it was a mistake to do such a thing, but disagree with your vision of the common interface. This interface should provide a way to access to multiple semantic owners.

Reply:
@dima-avdeev-jb 
Is my assumption correct, that ComposeSceneLayer will have 1-to-1 relationship with corresponding SemanticsOwner on iOS?

Reply:
I don't know exact SemanticsOwner behavior for now. But looks like - Yes.